### ISSUE TITLE
generate appears to be broken

### ISSUE THREAD

**Issue by gabeweaver** [apollostack/apollo-codegen#37]
Am I doing something wrong or are these errors able to be recreated by anyone else?

# Environment
* `nvm 0.32.1`
* `node 7.4.0`
* `yarn 0.18.1`
* `Darwin Kernel Version 16.3.0: Thu Nov 17 20:23:58 PST 2016; root:xnu-3789.31.2~1/RELEASE_X86_64 x86_64`

# Steps To Reproduce
* Install apollo-codegen via `yarn global add apollo-codegen`
* Run `apollo-codegen download-schema ...`
* Run `apollo-codegen generate **/*.graphql ...`

I was originally trying to generate flow types and wanted to verify undesired behavior so I attempted the same command so i tried with `--target typescript` instead of `--target flow`.  The inputs and outputs of both commands are below. 

# Errors
## Flow Scenario
Command: 
```
apollo-codegen generate **/*.graphql --schema schema.graphql.json --target flow --output schema.flow.js`
```
Response:
```
Invalid values:
  Argument: target, Given: "flow", Choices: "swift", "json", "ts", "typescript"
```

## TypeScript Scenario
Command: 
```
apollo-codegen generate **/*.graphql --schema schema.graphql.json --target typescript --output schema.graphql.ts
```
Response:
```
Error: ENOENT: no such file or directory, open '/Users/Gabe/Sites/foresight-client/**/*.graphql'
    at Object.fs.openSync (fs.js:558:18)
    at Object.fs.readFileSync (fs.js:468:33)
    at /Users/Gabe/.nvm/versions/node/v7.4.0/lib/node_modules/apollo-codegen/src/loading.js:28:21
    at Array.map (native)
    at loadAndMergeQueryDocuments (/Users/Gabe/.nvm/versions/node/v7.4.0/lib/node_modules/apollo-codegen/src/loading.js:27:30)
    at generate (/Users/Gabe/.nvm/versions/node/v7.4.0/lib/node_modules/apollo-codegen/src/generate.js:14:20)
    at Object.handler (/Users/Gabe/.nvm/versions/node/v7.4.0/lib/node_modules/apollo-codegen/src/cli.js:91:9)
    at Object.self.runCommand (/Users/Gabe/.nvm/versions/node/v7.4.0/lib/node_modules/apollo-codegen/node_modules/yargs/lib/command.js:170:22)
    at parseArgs (/Users/Gabe/.nvm/versions/node/v7.4.0/lib/node_modules/apollo-codegen/node_modules/yargs/yargs.js:920:28)
    at Object.get [as argv] (/Users/Gabe/.nvm/versions/node/v7.4.0/lib/node_modules/apollo-codegen/node_modules/yargs/yargs.js:860:16)
```

**Comment by marwanad** [comment]
AFAIK, the flow option was added after`v0.9.6` which isn't published to npm yet.

For the second error, it seems that your shell is misinterpreting `**/*.graphql`.

I assume `ls **/*.graphql` will also throw an error.

Can you try using that `$(find . -name "*graphql" -type f)` instead:
```
apollo-codegen generate $(find . -name "*graphql" -type f) --schema schema.graphql.json 
--target typescript --output schema.graphql.ts
```

**Comment by martijnwalraven** [comment]
@gabeweaver: This should be fixed in 0.9.7.
